lib/components/shorts/short_scroll_view.dart

```diff
@@ -135,6 +135,17 @@ class ShortScrollView extends HookConsumerWidget {
 
     final isRouteActive = useIsRouteActive();
     final isRouteActiveRef = useRef(isRouteActive);
+
+    useEffect(() {
+      if (isRouteActive) {
+        debugPrint('SHRT: enabling wakelock');
+        WakelockPlus.enable();
+      } else {
+        debugPrint('SHRT: disabling wakelock');
+        WakelockPlus.disable();
+      }
+    }, [isRouteActive]);
```

Comment:
This does the same as the one below (useValueChanged), except that it doesn't run on the initial value (only on change)
It's just me who forgot that detail in this commit: https://github.com/bcc-code/bcc-media-app/commit/35c5f6a2aca7913f017cac586a3e51cda47bfe9a#diff-aee3aac6f52e65bbb4c3cb9595b8607e2a60f0065de057373efdea57122bcfb4R134

So I would change it back to useEffect, to make sure the other code here like `tabOpenAnimation` runs.
Just make a variable for "wasActive" instead of using `useValueChanged`

(btw also noticed isFirstOpen is no longer in use)